octobot_trading/personal_data/exchange_personal_data.py

```diff
@@ -225,11 +227,13 @@ async def handle_closed_order_update(self, order_id, raw_order) -> bool:
             self.logger.exception(e, True, f"Failed to update order : {e}")
             return False
 
-    async def handle_trade_update(self, symbol, trade_id, trade, should_notify: bool = True):
+    async def handle_trade_update(self, symbol, trade_id, trade,
+                                  is_old_trade: bool = False, should_notify: bool = True):
         try:
             changed: bool = self.trades_manager.upsert_trade(trade_id, trade)
-            if should_notify:
-                await self.handle_trade_update_notification(trade, is_old_trade=False)
```

Comment:
`trade` was a dict here?

Reply:
When `handle_trade_update` is called from the trades channel, it is